about.html

```diff
@@ -9,65 +9,112 @@
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
 
+
+  <header>
+      <div class="topnav">
+          <img class= "a" src="lambda-black.jpg" width="10" height="10">
+          <a href="#home">Home</a>
```

Comment:
You almost had it! In order to link a file properly, you need to input the pathname of the file you want to link them to. 

Hint: look at line 98